### PR TITLE
Add nullability annotations to integrations modules

### PR DESCRIPTION
**Pull request by maciejwalkowiak**
## :scroll: Description

Add nullability annotations to integrations modules

## :bulb: Motivation and Context

Continuation of https://github.com/getsentry/sentry-java/pull/1439

## :pencil: Checklist

- [x] I reviewed the submitted code
- [ ] I added tests to verify the changes
- [ ] I updated the docs if needed
- [ ] No breaking changes

_#skip-changelog_

### CODE REVIEW

**Comment by maciejwalkowiak** [comment]
_#skip-changelog_

**Comment by marandaneto** [comment]
@maciejwalkowiak can you fix the conflicts and merge main into this branch so we can review? thanks

File: sentry/src/main/java/io/sentry/Session.java [hunk 1]
```diff
@@ -55,7 +55,7 @@
   private final @Nullable String environment;
 
   /** the App's release */
-  private final @NotNull String release;
+  private final @Nullable String release;
```
Comment (marandaneto) [inline sentry/src/main/java/io/sentry/Session.java:58]: cant be nullable

**Review by marandaneto — APPROVED** [review]
LGTM

**Pull request merged**